packages/ic-response-verification/src/error.rs

```diff
@@ -49,11 +49,15 @@ pub enum ResponseVerificationError {
 
     /// The tree has different root hash from the expected value in the certified variables
     #[error("Invalid tree root hash")]
-    InvalidTree,
-
-    /// The CEL expression path is invalid
-    #[error("Invalid expression path")]
-    InvalidExpressionPath,
+    InvalidTreeRootHash,
```

Comment:
The other change I've made in this PR is add more granular errors. This really helped me to debug this issue.